### PR TITLE
Add GeoParquet link to Data Catalog

### PR DESCRIPTION
**Pull request by Josmorsot**

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/utils/catalog/geoparquet.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export async function checkGeoparquetBucket (dataset) {
+  dataset = 'cdb_zcta5_fdb278bc';
```
Comment (Jesus89) [inline lib/assets/javascripts/new-dashboard/utils/catalog/geoparquet.js:2]: You need to remove this hardcoded slug

**Comment by Jesus89** [comment]
Linked to https://app.shortcut.com/cartoteam/story/347180/support-geoparquet-v1-0-in-carto

**Review by Jesus89 — APPROVED** [review]
LGTM

**Comment by jvillarf** [comment]
https://app.shortcut.com/cartoteam/story/347180/support-geoparquet-v1-0-in-carto

**Review by jvillarf — APPROVED** [review]

**Pull request merged**